interfaces/inc/file.h

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#ifndef SYNC_H
+#define SYNC_H
+
+#ifdef __cplusplus
+#include <cstdbool>
+#else
+#include <stdbool.h>
+#endif
+
+#include "umock_c/umock_c_prod.h"
+#include "constbuffer.h"
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define FILE_DATA_RESULT_VALUES \
+    FILE_DATA_FAILURE, \

```

Comment:
>FILE_DATA [](start = 4, length = 9)

Extra _DATA detected. Lt. Data, what are you doing here?

Reply:
This enum is used to indicate the success/failure of the file_read_async function. So the _DATA tag refers to the file data being read. tbh I just copied this over and tried to keep it same as much as possible. Can remove if you feel it is not needed.


---
In reply to: [445223096](https://github.com/Azure/azure-c-pal/pull/7#discussion_r445223096) [](ancestors = 445223096)